app/ui/src/app/import-export/import/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './import.component';
+export * from './import.actions';
+export * from './import.reducer';
+export * from './import.models';
+export * from './import.module';
```

Comment:
models, actions and reducer are most likely unnecessary outside the scope of this module, so there is no need to expose those.